src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy.mdx

```diff
@@ -136,3 +136,135 @@ The policies index lists policies in alphabetical order. To view or search for e
+  </Collapser>
+    <Collapser
+    id="delete-policy"
+    title="Delete a policy"
+  >
+    If you delete a policy, we'll continue to apply any other policies assigned to the product entity (target). If you remove **all** policies for the entity, its [color-coded health status](/docs/alerts/new-relic-alerts/configuring-alert-policies/identify-entities-without-alert-policies) indicator will appear light green with an icon.
+    
+    If you delete any of your account's policies, you can't restore them. Instead, you must re-create them if necessary.
+
+    To delete a policy completely:
+
+    1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, then click **Alert policies.**
+    2. From the policy row, click the delete <Icon name="fe-trash-2"/>
+       icon.
+
+    Any notification channels associated with a deleted or disabled policy will remain. You must [delete notification channels](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/delete-alert-notification-channels) or [remove them](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/add-or-remove-policy-channels) individually as necessary.
+
+    In a deleted policy, [incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) and [event violations](/docs/alerts/new-relic-alerts-beta/reviewing-events/review-events-across-products) are provided as a read only resource.
+  </Collapser>
+
+  <Collapser
+    id="targets-policy"
+    title="Check for entities (targets) assigned to a policy"
+  >
+    To check whether [an entity (target) has any policies assigned](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/identify-entities-without-alert-policies):
+
+    1. Go to the product (APM, browser, etc.).
+    2. From the selected product's index, check whether the entity's color-coded health status is light green with an icon. ![icon-alerts-q-green.png](./images/icon-alerts-q-green.png "Alerts v3: alert icons")
+
+    <Callout variant="tip">
+      **Exception:** Infrastructure does not use this feature.
+    </Callout>
+  </Collapser>
+
+  <Collapser
+    id="incidents"
+    title="Change how threshold violations roll up into incidents"
+  >
+    To select how [violations to your thresholds](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/specify-when-new-relic-creates-incidents) are grouped into incident records (by policy, by condition, or by target and condition):
+
+    1. In the **[one.newrelic.com](https://one.newrelic.com/ "Link opens in a new window.")** top nav, click **Alerts & AI**, click **Alert policies**, then **(select a policy)**.
+    2. Click <Icon name="fe-settings"/>
+       **Incident preference**.
+  </Collapser>
+
+  <Collapser
+    id="link-to-ai"
+    title="Link to Incident Intelligence"
```

Comment:
I don't know what is "Incident Intelligence" or why I might care to link to it. Maybe rename title to something like "Get context for alert violations with Incident Intelligence" ?